frontend/src/styles/_uswds-theme.scss

```diff
@@ -29,18 +28,36 @@ in the form $setting: value,
       "tablet": true,
       "tablet-lg": true,
       "desktop": true,
-      "desktop-lg": false,
+      "desktop-lg": true,
       "widescreen": false
     ),
+  $theme-banner-max-width: "desktop-lg",
+  $theme-header-max-width: "desktop-lg",
+  $theme-grid-container-max-width: "desktop-lg",
+  $theme-footer-max-width: "desktop-lg",
+  $theme-identifier-max-width: "desktop-lg",
+  $theme-column-gap-desktop: 6,
+  $theme-column-gap-md: 4,
+  $theme-column-gap-lg: 6,
```

Comment:
Increase the column gaps so we don't have to specify `"lg"` for every single one. Make large a bit larger too.